config/unit.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/gocty"
+)
+
+// Unit represents a list of units.
+type Unit struct {
+	Name   string `cty:"name"    hcl:",label"`
+	Source string `hcl:"source,attr" cty:"source"`
+	Path   string `hcl:"path,attr" cty:"path"`
+}
+
+// ToCtyValue converts StackConfigFile to cty.Value
+func (s *StackConfigFile) ToCtyValue() (cty.Value, error) {
```

Comment:
Do we need to have the file available as a cty value right now? We're going to integrate reference to the generating `terragrunt.stack.hcl` file in the future once we support a `values` attribute, but is this doing anything right now?

Reply:
Merged unit.go with stack.go since we only used unit structure from it.